### ISSUE TITLE
Blade style @apply directive doesn't work

### ISSUE THREAD

**Issue by aidilrx04** [tailwindlabs/tailwindcss#13316]
**What version of Tailwind CSS are you using?**

v3.4.1

**What build tool (or framework if it abstracts the build tool) are you using?**

laravel vite

**What version of Node.js are you using?**

v20.11.1

**What browser are you using?**

Chrome

**What operating system are you using?**

Windows

**Reproduction URL**

[https://github.com/aidilrx04/laravel-tailwind-apply-bug](https://github.com/aidilrx04/laravel-tailwind-apply-bug)
in file resource/views/welcome.blade.php

**Describe your issue**

The @apply directive doesn't generate/apply the style to the CSS selector using Blade template.

**Comment by aidilrx04** [comment]
Just found out that one way to solve this is through using @layer directive in app.css.

**Comment by adamwathan** [comment]
Hey! Tailwind only processes the CSS file you tell it to process here:

https://github.com/aidilrx04/laravel-tailwind-apply-bug/blob/main/vite.config.js#L7

It doesn't compile `<style>` tags in templates — to do that we'd have to overwrite the contents of your template file or build a copy of it and put it in a different folder somewhere.

TL;DR move that code to your actual CSS file that Tailwind compiles and you'll be good 👍
